src/main/kotlin/org/rust/ide/formatter/settings/RustLanguageCodeStyleSettingsProvider.kt

```diff
@@ -55,7 +55,11 @@ class RustLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider(
     override fun getDefaultCommonSettings(): CommonCodeStyleSettings =
         CommonCodeStyleSettings(language).apply {
             RIGHT_MARGIN = 99
-            initIndentOptions()
+            initIndentOptions().apply {
+                // FIXME(jajakobyly): It's a hack
+                // Nobody else does this and still somehow achieve similar effect
+                CONTINUATION_INDENT_SIZE = INDENT_SIZE
```

Comment:
Does anybody know how other plugins make continuation indents look like normal indents? I couldn't find how it's done in Go, Erlang, Java & Kotlin plugins (or maybe I just don't know where to search :sweat_smile: )